src/entities/inventory.entity.js

```diff
@@ -0,0 +1,13 @@
+class Inventory {
+  #id;
+  #productId;
+  #providerId;
+  #purchaseDate;
+  #totalArticlePurchased;
+  #totalPrice;
+  #individualArticlePrice;
+
+  getInventary = (purchaseDate) => {};
```

Comment:
I think you can remove these functions for now, they seemed to be confusing in this moment since the y are not doin anything right now.